packages/liveblocks-core/src/room.ts

```diff
@@ -671,18 +679,7 @@ type RoomState<
     storageOperations: Op[];
   };
 
-  /**
-   * Number of reconnect() retries.
-   */
-  numRetries: number;
-  readonly timers: {
-    flush: TimeoutID | undefined;
-    reconnect: TimeoutID | undefined;
-    heartbeat: IntervalID | undefined;
-    pongTimeout: TimeoutID | undefined;
-  };
-
-  readonly connection: ValueRef<Connection>;
+  readonly connection: ValueRef<Connection>; // TODO Make this a derived property?
```

Comment:
is this TODO something you meant to be part of this merge or further down the road?

Reply:
It's something I wanted to initially make part of this work before the scope grew. At this point, I'm happy to leave it and tackle it in a future change.